validator/validator-main.protoascii

```diff
@@ -3492,6 +3492,28 @@ tags: {
   }
 }
 
+# Fixes a visual bug when amp-story-1.0.css is Server Side Rendered.
```

Comment:
Nit: I think we should call it "dvh polyfill"

Reply:
@gmajoulet should i rename the attribute too you think?

Reply:
ill go ahead and update it